dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS/Utilities.pm

```diff
@@ -447,6 +447,7 @@ sub standard_XS_defs {
 EOF
 
   print <<"EOF";
+#ifndef croak_xs_usage
 #ifndef PERL_ARGS_ASSERT_CROAK_XS_USAGE
```

Comment:
I don't think this double `#ifndef` is needed, `#ifndef croak_xs_usage` should be enough, right?

Reply:
I think you are right, this was more precaution than anything else
I also think it makes it very clear and it s good habit to check if something is defined before defining it.
The line just after does a `#define PERL_ARGS_ASSERT_CROAK_XS_USAGE`